components/collection/drop/modal/paid/MintOverview.vue

```diff
@@ -13,13 +13,15 @@
           <div class="relative">
             <NeoSkeleton
               v-if="!toMintNft.imageDataPayload"
-              class="absolute border border-border-color overflow-hidden"
-              :width="48"
-              :height="48"
+              class="border border-border-color overflow-hidden"
+              :class="{ absolute: toMintNft.canRender }"
+              :width="46"
+              :height="46"
               :rounded="false"
               no-margin />
 
             <BaseMediaItem
+              v-if="toMintNft.canRender"
               :src="sanitizeIpfsUrl(toMintNft.image)"
               :alt="toMintNft.name"
               mime-type="text/html"
```

Comment:
Do we have a placeholder image if not rendering arts?

Reply:
a skeleton is shown if the NFT is not rendered 

![CleanShot 2024-03-28 at 13 26 08@2x](https://github.com/kodadot/nft-gallery/assets/44554284/b6f98e02-3ca6-46c6-b9ff-0fbdaf8579d4)

Reply:
Maybe the skeleton would look like it's still loading.

Reply:
ui wise nothing has changed, we still show all NFTs to be minted , but the amount of canvas that are rendered on the first load is what changed

just realized maybe the naming is confusing`MAX_RENDER_AT_ONCE_AMOUNT` 

first batch of 3 starting to render 
![CleanShot 2024-03-28 at 13 38 22@2x](https://github.com/kodadot/nft-gallery/assets/44554284/ade0035e-c002-4da6-8cba-56e4c32ff40e)

once one is completed we start rendering another one , basically we add another canvas.

![CleanShot 2024-03-28 at 13 41 17@2x](https://github.com/kodadot/nft-gallery/assets/44554284/5399f52c-50a0-41d3-851c-04d4e35e0fa3)


![CleanShot 2024-03-28 at 13 33 55](https://github.com/kodadot/nft-gallery/assets/44554284/04449713-5a09-433b-a20c-74bdd6a3fd65)

Reply:
It looks good. Thanks for providing the screenshots.